### PR TITLE
test(cdr-modal tests): refactor cedar modal tests

### PR DESCRIPTION
**Pull request by KenjiCrosland**
CDR-2541

### CODE REVIEW

File: src/components/modal/__tests__/__snapshots__/CdrModal.spec.js.snap [hunk 1]
```diff
@@ -73,7 +73,7 @@ exports[`CdrModal.vue > default open 1`] = `
             <div
               class="cdr-modal__text-content"
               role="document"
-              style="max-height: -80px; padding-right: 0px;"
```
Comment (KenjiCrosland) [inline src/components/modal/__tests__/__snapshots__/CdrModal.spec.js.snap:76]: @benjag this looks right? We should see a positive value for the opened state correct?

Also, maybe we should have another snapshot for a closed state?

Comment (benjag) [reply src/components/modal/__tests__/__snapshots__/CdrModal.spec.js.snap:76]: Ok, I expect you're right on the nextTick() being too fast then

Comment (KenjiCrosland) [reply src/components/modal/__tests__/__snapshots__/CdrModal.spec.js.snap:76]: I think this is good to go. I added another snapshot for the closed state.

File: src/components/modal/__tests__/CdrModal.spec.js [hunk 2]
```diff
@@ -26,6 +26,10 @@ describe('CdrModal.vue', () => {
       expect(wrapper.element).toMatchSnapshot();
```
Comment (KenjiCrosland) [inline src/components/modal/__tests__/CdrModal.spec.js:26]: @benjag I noticed these tests were commented out and your name was on the blame. I re-enabled them in this PR, fyi

**Review by benjag — APPROVED** [review]

**Pull request merged**